pgjdbc/src/main/java/software/aws/rds/jdbc/postgresql/auth/AwsIamAuthenticationPlugin.java

```diff
@@ -1,17 +1,16 @@
 package software.aws.rds.jdbc.postgresql.auth;
 
-import org.checkerframework.checker.nullness.qual.Nullable;
-
-import software.amazon.awssdk.auth.credentials.DefaultCredentialsProvider;
-import software.amazon.awssdk.regions.Region;
-import software.amazon.awssdk.services.rds.RdsUtilities;
-
 import org.postgresql.PGProperty;
 import org.postgresql.plugin.AuthenticationPlugin;
 import org.postgresql.plugin.AuthenticationRequestType;
 import org.postgresql.util.GT;
 import org.postgresql.util.PSQLException;
 
+import org.checkerframework.checker.nullness.qual.Nullable;
```

Comment:
Is this the correct order of imports? Can you check if the project has guidelines about import order?